provider/parameter_test.go

```diff
@@ -686,9 +689,9 @@ data "coder_parameter" "region" {
 func TestValueValidatesType(t *testing.T) {
 	t.Parallel()
 	for _, tc := range []struct {
-		Name,
-		Type,
-		Value,
+		Name  string
+		Type  provider.OptionType
+		Value string
 		Regex,
 		RegexError string
 		Min,
```

Comment:
TIL you can elide these. If we re-arrange the struct fields does that mean we can just elide the type of `RegexError` as well?

e.g.

```
Type provider.OptionType
Name string
Value,
Regex,
RegexError,
Min, ...
```